lend/src/components/StepsHeader/index.tsx

```diff
@@ -168,11 +166,12 @@ const StepsHeader: React.FC = () => {
         </AnimatePresence>
       </StepsHeaderTextContainer>
       <ButtonContainer>
-        <a href={URLS.lendApp}>
-          <OpenAppButton>
-            <ButtonText>OPEN APP</ButtonText>
-          </OpenAppButton>
-        </a>
+        {/* uncomment on launch */}
+        {/* <a href={URLS.lendApp}> */}
+        <OpenAppButton>
+          <ButtonText>OPEN APP</ButtonText>
```

Comment:
Should we not hide this for the timebeing?

Reply:
technically its hiding already seems fine